### ISSUE TITLE
Audit: PDM-004 - Vulnerable and Unmaintained Dependencies

### ISSUE THREAD

**Issue by ashneverdawn** [pendulum-chain/pendulum#266]

**Comment by vadaynujra** [comment]
Hey team! Please [add your planning poker estimate](https://app.zenhub.com/workspaces/Pendulum-6408a0365cda59c57d39bee5/issues/pendulum-chain/pendulum/266?planning-poker) with Zenhub @ashneverdawn @ebma @TorstenStueber

**Comment by ebma** [comment]
Not sure what the immediate actions for this issue would be. 
Maybe #150 is related and would kind of also close this issue ticket?

**Comment by vadaynujra** [comment]
@ashneverdawn what do you think?

**Comment by ashneverdawn** [comment]
I think the Remediation column shows what the actions should be taken, right? Looks like it's mainly just updating dependencies. I think #150 was only to enable alerts to detect these kinds of issues, not actually solve them.

**Comment by TorstenStueber** [comment]
Yes, I think #150 will not solve it but will make it easier for us to detect it ourselves in the future.

**Comment by TorstenStueber** [comment]
However, the recommendation for some crates here is to switch to alternative crates and that could be quite hard to do if it is transitive dependencies. Let's see.

**Comment by gianfra-t** [comment]
The dependencies that raise concern are indirectly imported into the project by substrate modules many levels deep. For instance, `parity-wasm` is required by `cumulus-pallet-parachain-system `  indirectly, and so on. 
The only remediation possible is to try and update to the latest branches of parity's modules, as soon as other modules like `orml-` pallet's also update, since we will indirectly also depend on them.

Even then, as of today, some unmaintained dependencies are still used in the newest [polkadot-sdk](https://github.com/paritytech/polkadot-sdk) , like `ansi-term`.

So in summary, trying to update to the latest parity's dependencies is the only action we could take for this issue.

**Comment by ebma** [comment]
@TorstenStueber and @pendulum-chain/product I would say we close this ticket, concluding that we cannot do much about it besides regularly updating the substrate dependencies (as is also mentioned in the recommendation of the auditors).
